src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBArgumentCollection.java

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.hellbender.tools.genomicsdb;
+
+import org.broadinstitute.barclay.argparser.Advanced;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.GenotypeCalculationArgumentCollection;
+
+public class GenomicsDBArgumentCollection extends GenomicsDBBaseArgumentCollection {
+  private static final long serialVersionUID = 1L;
+
+  private static final String MAX_GENOTYPE_COUNT_LONG_NAME = "genomicsdb-max-genotype-count";
+
+  @Advanced
+  @Argument(fullName=MAX_GENOTYPE_COUNT_LONG_NAME, doc="Maximum number of genotypes to consider at any site", optional=true)
```

Comment:
Since this is only used outside outside of the genotyping tools, I think it would make sense to remove the word "consider" from the doc string, and also reinforce that this only applies when the underlying store is GenomicsDB. Maybe something like "Maximum number of genotypes to retrieve when the underlying store is GenomicsDB".

Reply:
This argument was being used only by SelectVariants. Note that the existing `GenomicsDBArgumentCollection` is removed and `GenomicsDBBaseArgumentCollection` is renamed to `GenomicsDBArgumentCollection`.